packages/shared/components/atoms/labels/AccountLabel.svelte

```diff
@@ -1,17 +1,17 @@
 <script lang="typescript">
+    import { IAccountState } from '@core/account'
     import { Text } from 'shared/components/'
     import { FontWeightText } from 'shared/components/Text.svelte'
-    import { IAccountState } from '@core/account'
 
     export let account: IAccountState
     export let selected = true
 </script>
 
-<div class="flex flex row items-center space-x-3">
+<div class="flex row items-center space-x-3">
```

Comment:
Looks like this class doesn't exist
```suggestion
<div class="flex items-center space-x-3">
```